2016/playbook/session_2/SCHEDULE.md

```diff
@@ -81,7 +81,23 @@
 
 ## Day 4
 
-On day 4 we'll be running the [Dodge][dodge] workshop.
+| Time     | Activity                                           |
+| -------- | -------------------------------------------------- |
+| 8:40 AM  | Standups                                           |
+| 9:00 AM  | Hacker arrival                                     |
+| 9:10 AM  | Energizer 1                                        |
+| 9:25 AM  | Breakfast                                          |
+| 9:45 AM  | [Dodge][dodge]          						    |
+| 12:30 PM | Lunch                                              |
+| 1:30 PM  | Energizer 2                                        |
+| 2:00 PM  | [Dodge (continued)][dodge] 						|
+| 3:30 PM  | [Energizer 3](../ACTIVITIES.md#walking)            |
+| 3:45 PM  | [Dodge (continued)][dodge] 						|
```

Comment:
Sorta strange because it's formatted all correctly on mine... Let me commit my changes for ACTIVITES.md and see if that affects it in any way